OSDOCS-9559: Added OIDC provider note to prereqs (#71488)
openshift/openshift-docs: rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

Before running the installation process, verify that you deploy this from a machine that has access to:

* The API services for the cloud to which you provision.
* Access to `api.openshift.com`, `oidc.op1.openshiftapps.com`, and `sso.redhat.com`.
* The hosts on the network that you provision.
* The internet to obtain installation media.

[IMPORTANT]
====

> Starting with ROSA CLI version 1.2.7, all new cluster OIDC provider endpoint URLs are no longer regional and instead use the link:http://oidc.op1.openshiftapps.com/[oidc.op1.openshiftapps.com] domain. Amazon CloudFront is part of this implementation to improve access speed, reduce latency, and improve resiliency. This change is only available for new clusters created with ROSA CLI 1.2.7 or later. There are no supported migration paths for existing OIDC provider configurations.

Reviewer: ```suggestion
Starting with version 1.2.7 of the ROSA CLI, all OIDC provider endpoint URLs on new clusters use Amazon CloudFront and the link:http://oidc.op1.openshiftapps.com/[oidc.op1.openshiftapps.com] domain. This change improves access speed, reduces latency, and improves resiliency for new clusters created with the ROSA CLI 1.2.7 or later. There are no supported migration paths for existing OIDC provider configurations.
```
